### DISCUSSION TITLE
DC-ROMA%20II

### DISCUSSION THREAD

**Discussion by giscus** [fedora-riscv/fedorav-force-images-discussions#5] [Q&A]
# DC-ROMA%20II

https://images.fedoravforce.org/DC-ROMA%20II

**Comment by imguoguo** [comment]
The front camera device is located at /dev/video1. The Camera app is unavailable, but the device is functioning normally. Testing on webcamtests.com in Firefox works fine.

**Reply by imguoguo** [reply]
Fixed

**Comment by glen102** [comment]
Great to see a new image thanks! This works really well off the SD card in my computer but I also have an NVME drive (currently running the latest Bianbu). Do you have any way to put the images of Fedora onto and boot from the NVME drive in the DC-ROMA 2?

**Reply by imguoguo** [reply]
Yes, this is a known issue. The onboard firmware of the DC ROMA 2 currently does not support the newer kernel versions. Additionally, when booting from NVMe, the computer will not read the firmware from the NVMe drive. Therefore, it is necessary to upgrade the onboard firmware before it can be used. Documentation for the firmware upgrade will be organized and provided shortly.
